app/memory/chroma_memory.py

```diff
@@ -33,20 +42,25 @@ def __init__(self, **kwargs: Any):
         embeddings = EmbeddingsModel(**kwargs)
 
         persist_directory = os.path.join(
-            os.getcwd(), "data", config.memories_path)
+            os.getcwd(), "data", config.memories_path
+        )
         # Create directory if it doesn't exist
         os.makedirs(persist_directory, exist_ok=True)
 
         self.vector_store = vectorstores.Chroma(
             collection_name=self.collection_name,
             embedding_function=embeddings,
-            persist_directory=persist_directory
+            persist_directory=persist_directory,
         )
 
         self.setup_index()
 
     def setup_index(self):
-        collection = self.vector_store._client.get_collection(self.collection_name)
+        collection = self.vector_store._client.get_collection(
+            self.collection_name
+        )
         if len(collection.get()['ids']) < 6:
-            self.add_texts(["Hello", "world!", "this", "is", "a", "test"], ids=[
-                           "1", "2", "3", "4", "5", "6"])
+            self.add_texts(
+                ["Hello", "world!", "this", "is", "a", "test"],
+                ids=["1", "2", "3", "4", "5", "6"],
+            )
```

Comment:
Why do we have this block BTW? Could we remove it, or is it somehow still useful?

Reply:
without it, when you query the index with `k=x` an error is raised if you do not have at least `x` documents in the vectordb. There is surely a better way to handle this. Let's table it for another PR.